<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="matDisplay" name="mat_Display">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>540</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>540</width>
      <height>500</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Eventos</string>
    </property>
    <widget class="QLabel" name="mouse_position_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pixel_color_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="colored_label">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buton_load_image">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar Imagem</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuComputa_o_Gr_fica">
    <property name="title">
     <string>Imagem</string>
    </property>
    <addaction name="actionPreto_e_Branco"/>
    <addaction name="actionNegativa"/>
   </widget>
   <widget class="QMenu" name="menuDesenhar">
    <property name="title">
     <string>Desenhar</string>
    </property>
    <addaction name="actionPixel"/>
    <addaction name="actionReta"/>
    <addaction name="actionAlgoritmo_de_Bresenham"/>
    <addaction name="separator"/>
    <addaction name="actionC_rculo_y_sqrt_R_R_x_x"/>
    <addaction name="actionCirculo_Equa_o_param_trica"/>
   </widget>
   <addaction name="menuComputa_o_Gr_fica"/>
   <addaction name="menuDesenhar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPreto_e_Branco">
   <property name="text">
    <string>Preto e Branco</string>
   </property>
  </action>
  <action name="actionNegativa">
   <property name="text">
    <string>Negativa</string>
   </property>
  </action>
  <action name="actionPixel">
   <property name="text">
    <string>Pixel</string>
   </property>
  </action>
  <action name="actionReta">
   <property name="text">
    <string>Reta (y = ax + b)</string>
   </property>
  </action>
  <action name="actionAlgoritmo_de_Bresenham">
   <property name="text">
    <string>Algoritmo de Bresenham</string>
   </property>
  </action>
  <action name="actionC_rculo_y_sqrt_R_R_x_x">
   <property name="text">
    <string>Círculo (y = sqrt(R*R – x *x))</string>
   </property>
  </action>
  <action name="actionCirculo_Equa_o_param_trica">
   <property name="text">
    <string>Circulo (Equação paramétrica)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matDisplay</class>
   <extends>QLabel</extends>
   <header>matdisplay.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
